Makefile

```diff
@@ -63,6 +64,11 @@ check_third_party_binary:
 	@which bin/sync_diff_inspector
 	@which bin/go-ycsb
 
+integration_test_build:
+	$(GOTEST) -c -cover -covemode=atomic \
```

Comment:
Why is `$(GOTEST)` called integration test here?

Reply:
Here we use `go test` tool to compile a `cdc binary`, which compiles the package in `main_test.go` and links to the main test program in `main_test.go`. In this way we can run a cdc daemon in the `go test` way.

Reply:
Maybe add an example of how to use it in the comment?

Reply:
already have a reference link in `main_test.go`